cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -82,36 +111,38 @@ metadata:
   name: token-${safe_username}
 type: Opaque
 EOF
-  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" &
+  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" "${SYSTEM_NAMESPACE}" &
```

Comment:
Do you want me to remove the comment, do another rebase and push?
Or deal with it in a suibsequent PR? There are still some other minor
things that need to be dealt with (e.g. some extra whitespace).
However, out of paranoia, let me point out that the currently pushed
version of this PR has had several good local e2e runs on my desktop
machine.

On Mon, Jul 6, 2015 at 10:17 AM, Zach Loafman <notifications@github.com>
wrote:

> In cluster/saltbase/salt/kube-addons/kube-addons.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10487#discussion_r33958104>
> :
>
> > @@ -82,36 +111,38 @@ metadata:
> >    name: token-${safe_username}
> >  type: Opaque
> >  EOF
> > -  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" &
> > +  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" "${SYSTEM_NAMESPACE}" &
>
> I believe this comment is no longer applicable. (Just documenting in case
> someone is looking at the same view I am).
>
> We no longer handle any updates in this script, so we're just creating the
> kube-system token here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10487/files#r33958104>
> .
>